website/content/commands/connect/envoy.mdx

```diff
@@ -26,6 +28,27 @@ file and used with `envoy --config-path bootstrap.json`. This works on all opera
 systems allowing configuration to be generated on a host that Envoy doesn't
 build on but then used in a virtualized environment that can run Envoy.
 
+This command has the following characteristics:
+
+| Characteristic | Value |
+| -------------- | ----- |
+| [Required ACLs] | Varies<sup>1</sup> |
+| Corresponding HTTP API endpoint | N/A |
+
+[Service identity](/docs/security/acl#service-identities) for the [service this proxy represents](#service)
+
+<sup>1</sup> The actual ACL required depends on the whether the Envoy instance is acting
+[as a sidecar proxy](#sidecar-for) or
+[as a gateway](#gateway).
+
+| Envoy proxy type       | Required ACL       |
+| ---------------------- | ------------------ |
+| Sidecar proxy          | [Service identity](/docs/security/acl#service-identities) for the [service name this proxy represents](#service)    |
+| Ingress gateway        | Refer to [Ingress Gateway Configuration](/docs/connect/gateways/ingress-gateway#ingress-gateway-configuration) |
+| Terminating gateway    | Refer to [Terminating Gateway Configuration](/docs/connect/gateways/terminating-gateway#terminating-gateway-configuration) |
+| Mesh gateway           | Refer to [Mesh Gateway Configuration](/docs/connect/gateways/mesh-gateway/service-to-service-traffic-datacenters#configuration) |
+| API gateway (k8s only) | Handled automatically if [`managedSystemACLs: true`](/docs/k8s/helm#v-global-acls-managesystemacls) |
```

Comment:
@Jeff-Apple : can you review this line regarding the required ACL for the Consul API Gateway? I want to make sure it's accurate.

Reply:
@jkirschner-hashicorp, A user should never run this command against an Ingress Gateway created by Consul API Gateway (CAPIgw) because the CAPIgw Controller manages the bootstrapping of Envoy for those gateways. If a user does make a change with this command, it will either have no effect and/or be overwritten as soon as the CAPIgw Controller detected the change.